base/src/com/google/idea/blaze/base/qsync/BazelDependencyBuilder.java

```diff
@@ -162,91 +173,134 @@ public OutputInfo build(
     Optional<BuildDepsStats.Builder> buildDepsStatsBuilder =
         BuildDepsStatsScope.fromContext(context);
     buildDepsStatsBuilder.ifPresent(stats -> stats.setBlazeBinaryType(invoker.getType()));
-
-    String includes =
-        projectDefinition.projectIncludes().stream()
-            .map(path -> "//" + path)
-            .collect(joining(","));
```

Comment:
@odisseus you are right, they are dropping the comma separated collection. In this case I would suggest to drop all my changes and use the AOSP implementation.